Dockerfile

```diff
@@ -19,18 +17,16 @@ RUN apt-get update --quiet \
    && apt-get clean --quiet \
    && rm -rf /var/lib/apt/lists/*
 
-ARG APP_DIR
-ARG LOCAL_DIR
+ARG LOCAL_DIR=./submissions-pilot2
+ARG APP_DIR=/usr/local/src/submissions-pilot2
 
 COPY $LOCAL_DIR $APP_DIR
 
 WORKDIR $APP_DIR
 
-ARG RENV_VERSION
-
 RUN Rscript -e "renv::restore()"
 
-ARG R_SCRIPT
+ARG R_SCRIPT=./docker_app.R
```

Comment:
WDYT about creating a .Rprofile file instead of a `docker_app.R` file to set the options and additionally we can have the pkgload and app run as steps in the Dockerfile?

Mainly because this seems to be a standard that other packages follow (for example [using renv with docker](https://rstudio.github.io/renv/articles/docker.html)) and having a .R script suggests to me that it's a regular RScript while this is clearly not.

Reply:
I'm not a big fan because there's a risk of replacing existing `.Rprofile` that could have custom logic / helpers / something

This is more analogous to the `app.R` file in Rhino, what do you think of renaming it to `app.R`?

Or `entrypoint.R` / `cmd.R` which would not conflict with other files and match Dockerfile vocabulary

Reply:
I'm not a big fan of that, because it will create a hidden file in Linux / Mac OSX systems. 

And copying it directly to the container might overwrite custom logic / helpers in the existing `.Rprofile`. 

How about keeping it as `entrypoint.R` or `cmd.R` ? That's in line with Docker vocabulary

Reply:
I see your point. I think `entrypoint.R` seems like a good name 👍🏽

Reply:
@vedhav I've changed it, let me know if you have any other comment :)

Reply:
Thanks, I'm trying to restore this on my mac but it does not seem to work. I'm trying to get to the root of this. The problem is with installing the `formatters` package.
Also, all the package "Repository" in the renv.lock file says `RSPM` while the renv.lock file only has `CRAN` and `TEAL` Repositories present. Am I missing something, How does this work?

Reply:
Oh I see, In case the package is not found it tries to loop through other versions of the repository mirrors we've listed. And `formatters` seems to be installed from the depository.

Reply:
Good catch, beats me! The _"Repository"_ option might only be a suggestion and `renv` looks in all repositories in case it doesn't find "RSPM"